examples/posenet/Drawoutput.js

```diff
@@ -0,0 +1,81 @@
+const color = 'aqua';
+const lineWidth = 2;
+
+
+function loadImage(imagePath, ctx) {
+  const image = new Image();
+  const promise = new Promise((resolve, reject) => {
+    image.crossOrigin = '';
+    image.onload = () => {
+    	ctx.drawImage(image, 0, 0);
+      console.log("image width: ", image.width);
+      resolve(image);
+    };
+  });
+
+  image.src = imagePath;
+  return promise;
+}
+
+function eitherPointDoesntMeetConfidence(a, b, minConfidence) {
+    return (a < minConfidence || b < minConfidence);
```

Comment:
Keep the indention consistent to 2-spaces. Change the 4-spaces indention to 2-spaces throughout all files.